content/en/docs/reference/command-line-tools-reference/kube-apiserver.md

```diff
@@ -913,7 +913,7 @@ kube-apiserver [flags]
       <td colspan="2">--shutdown-delay-duration duration</td>
     </tr>
     <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">Time to delay the termination. During that time the server keeps serving requests normally and /healthz returns success, but /ready immediately returns failure. Graceful termination starts after this delay has elapsed. This can be used to allow load balancer to stop sending traffic to this server.</td>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">Time to delay the termination. During that time the server keeps serving requests normally and /healthz returns success, but /readzy immediately returns failure. Graceful termination starts after this delay has elapsed. This can be used to allow load balancer to stop sending traffic to this server.</td>
```

Comment:
`readzy` looks like a typo; does that need fixing upstream?

Reply:
Turns out that, actually, @sftim already did.
@daminisatya did you have revision https://github.com/kubernetes/kubernetes/commit/70132b0f130acc0bed193d9ba59dd186f0e634cf checked out when you generated the reference documentation?

Reply:
If I clone `kubernetes/kubernetes` and check out release-1.17 then run:
```bash
git annotate ./staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go | grep "returns success, but /r.*"
```
I see the fixed wording with `/readyz`.

Puzzled.

Reply:
And ... it was fixed by you, @sftim :)

https://github.com/kubernetes/kubernetes/commit/bbccdcd8531a6377c1fe2937aef24f61caef43c4